src/org/parosproxy/paros/control/Control.java

```diff
@@ -546,8 +546,10 @@ public Mode getMode() {
 		return mode;
 	}
 	public void setMode(Mode mode) {
-		this.mode = mode;
-		getExtensionLoader().sessionModeChangedAllPlugin(mode);
-		model.getOptionsParam().getViewParam().setMode(mode.name());
+		if (!mode.name().equals(model.getOptionsParam().getViewParam().getMode())) {

```

Comment:
https://status.github.com/
> 13:22 Eastern Standard Time
We're monitoring a delay in job processing. Some events may not be reflected immediately.